code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4448,7 +4448,10 @@
 	..()
 	reagents.add_reagent(NUTRIMENT, 3)
 	reagents.add_reagent(SUGAR, 2)
-	icon_state = "sweet[rand(1,12)]"
+	var/list/flavors = list("\improper strawberry","\improper lime","\improper blueberry","\improper lemon","\improper grape","\improper lemonade","\improper bubblegum","\improper raspberry","\improper orange","\improper liquorish","\improper apple","\improper cranberry")
+	var/variety = rand(1,12)
```

Comment:
I'd say make it variety = rand(1, flavors.len)

Reply:
Sure, coming right up.